.travis.yml

```diff
@@ -1,40 +1,33 @@
 language: ruby
 cache: bundler
 rvm:
-  - 2.2.10
-  - 2.3.7
   - 2.4.4
-  - 2.5.1
-  - jruby-9.1.17.0
+  - 2.5.3
   - jruby-9.2.0.0
+  - jruby-9.2.5.0
```

Comment:
might as well use the latest version 9.2.7.0

Reply:
updated it